core/wiki/macros/toolbar.tid

```diff
@@ -0,0 +1,73 @@
+title: $:/core/macros/toolbar
+tags: $:/tags/Macro
+
+\define toolbar-filter-display(tag, config)
```

Comment:
The coding style for the core is not to insert spaces after commas in parameter lists